.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: build
+on:
+  push:
+    branches:
+      - '*'
+  pull_request: {}
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
```

Comment:
I think sam had requested we use v4.. I've done that for the observability-api actions as well.